ui/package.json

```diff
@@ -96,7 +96,7 @@
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-maybe-in-element": "^0.1.3",
-    "ember-power-select": "^2.0.12",
+    "ember-power-select-with-create": "cibernox/ember-power-select-with-create#6203918f247c1c5d692db4f9185ab8321d2125e1",
```

Comment:
Can we lock to a version instead of a sha? Maybe try 0.6.0 if 0.6.1 was a problem. What was the issue with the tagged 0.6.1?

Reply:
Chatted about this and it's because we need `placeholderComponent` which was added https://github.com/cibernox/ember-power-select-with-create/pull/82 but there's no release with that yet.